android/res/xml/settings_application.xml

```diff
@@ -18,35 +18,56 @@
  */
 -->
 <PreferenceScreen xmlns:android="http://schemas.android.com/apk/res/android">
+    <PreferenceCategory android:key="frostwire.prefs.general"
```

Comment:
I think it's useful to get a hold of the category in java-land, at least before material design

```java
    private void setupTorrentMaxDownloadSpeed(final BTEngine e) {
        NumberPickerPreference pickerPref = (NumberPickerPreference) findPreference(Constants.PREF_KEY_TORRENT_MAX_DOWNLOAD_SPEED);
...
```